### PR TITLE
Add insights.ubuntu.com

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added insights.ubuntu.com

## QA

``` bash
# Install VirtualBox if you haven't got it
sudo apt install virtualbox

# Install minikube if you haven't already
curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.22.3/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/

# Enable Ingress add-on
minikube addons enable ingress

# Get minkube running
minikube start
kubectl config set-context minikube  # To be on the safe side

# Deploy new configs to minikube
cat services/insights.ubuntu.com.yaml | sed 's|prod-comms.docker-registry.canonical.com/insights.ubuntu.com|canonicalwebteam/www.ubuntu.com|' | sed 's|[:][$][{]TAG_TO_DEPLOY[}]||' | sed 's|replicas: 5|replicas: 1|' | kubectl apply --namespace staging --filename -
cat services/insights.ubuntu.com.yaml | sed 's|prod-comms.docker-registry.canonical.com/insights.ubuntu.com|canonicalwebteam/www.ubuntu.com|' | sed 's|[:][$][{]TAG_TO_DEPLOY[}]||' | sed 's|replicas: 5|replicas: 1|' | kubectl apply --namespace production --filename -
kubectl apply --namespace staging --filename ingresses/staging/insights.staging.ubuntu.com.yaml
kubectl apply --namespace production --filename ingresses/production/insights.ubuntu.com.yaml

# Check everything deployed correctly
kubectl get ingress,service,deployment,pod --namespace staging  # Inspect staging elements
kubectl get ingress,service,deployment,pod --namespace production  # Inspect production elements

# Curl the site, check for 200 status
curl -I -H 'Host: insights.ubuntu.com' `minikube ip`
curl -I -H 'Host: insights.staging.ubuntu.com' `minikube ip`
```

### CODE REVIEW

File: ingresses/staging/insights.staging.ubuntu.com.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - secretName: staging-insights-ubuntu-com-tls
+    hosts:
+    - insights.staging.ubuntu.com
+  rules:
+  - host: insights.staging.ubuntu.com
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: insights.staging.ubuntu.com
```
Comment (nottrobin) [inline ingresses/staging/insights.staging.ubuntu.com.yaml]: This needs to by hyphens, not dots

File: ingresses/staging/insights.staging.ubuntu.com.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+
+kind: Ingress
+apiVersion: extensions/v1beta1
+metadata:
+  name: staging-insights-ubuntu-com
```
Comment (nottrobin) [inline ingresses/staging/insights.staging.ubuntu.com.yaml]: this should be "insights-staging-ubuntu-com"

File: ingresses/staging/insights.staging.ubuntu.com.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  namespace: staging
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/configuration-snippet: |
+      more_set_headers "X-Robots-Tag: noindex";
+spec:
+  tls:
+  - secretName: staging-insights-ubuntu-com-tls
```
Comment (nottrobin) [inline ingresses/staging/insights.staging.ubuntu.com.yaml]: This should be "insights-staging-ubuntu-com"

**Comment by nottrobin** [comment]
Fixes https://github.com/canonical-websites/insights.ubuntu.com/issues/42

**Comment by nottrobin** [comment]
@anthonydillon I assume those problems are reflective of problems with the script you wrote? Maybe update it?

**Comment by anthonydillon** [comment]
They are problems that were find when you QA'ed the script. The script was initially based on this branch but include these fixes.

**Comment by anthonydillon** [comment]
@nottrobin could you update the labels please.

**Pull request merged**